modules/wsl-distro.nix

```diff
@@ -102,20 +103,16 @@ with lib; {
             '';
           };
 
+          # no udev devices can be attached
+          services.udev.enable = false;
```

Comment:
Could we at least make this overrideable (`lib.mkDefault`) ?
Have the case in mind that someone tries to attach a usb scanner - did that once in a ubuntu wsl instance so should be possible at least.

Reply:
You can always overwrite it with lib.mkForce.

Reply:
That's true, but I'm in favor of using mkDefault as well. We're disabling udev by default, because it doesn't need to run; disabling it is not necessary for the system to work